_posts/2023-01-04-trino-2022-the-rabbit-reflects.md

```diff
@@ -0,0 +1,581 @@
+ critical for positioning Trino as a data lakehouse query engine. `MERGE` is 
+ currently available in Hive, Iceberg, Delta Lake, Kudu, and Raptor connectors.
+ We discussed this and did a demo with `MERGE` on the recent [Trino Community
+ Broadcast with Iceberg]({{site.url}}/episodes/40.html).
+* Another massive update was the introduction of [polymorphic table functions
+ ]({% post_url 2022-07-22-polymorphic-table-functions %}) ([PTFs
+ ]({{site.url}}/docs/current/functions/table.html)). Table functions
+ initially released with some initial passthrough query functionality that we
+ see in connectors like Pinot, Elasticsearch, MySQL, PostgreSQL, and many
+ others. However, this is only one small instance of what can be achieved with
+ PTFs and the [true power comes from the generalization of this feature
+ ](https://www.youtube.com/clip/UgkxQcokpdgPjiuMKMC5-3HwHvlbmZjxAvxe). Dain and
+ David gave [a simpler explanation of PTFs
+ ](https://www.youtube.com/clip/Ugkx62IKgPd_v9eGBaPUHP2hyaRkWSXh8w8h) if it’s
+ still not very clear what they are. To dive in deeper, watch [this episode of
+ the Trino Community Broadcast]({{site.url}}/episodes/38.html) where Kasia
+ Findeisen and Martin discuss PTFs in greater detail.
+* [Dynamic function resolution](https://github.com/trinodb/trino/issues/8) has
+ been discussed for many years. This provides the ability for [connectors to
+ provide functions at runtime](https://youtu.be/mUq_h3oArp4?t=680). Unlike
+ before, where you needed to statically register your functions ahead of time,
+ you can now provide a plugin that defines these functions that gets resolves
+ while Trino is running. This enables features like supporting function calls to
+ dynamically registered user-defined functions in different languages like
+ Javascript or Python. Martin and Dain go into great detail about how this works
+ when [answering this question at Trino Summit
+ ](https://youtu.be/mUq_h3oArp4?t=1596).
+* [ANSI SQL 2016](https://en.wikipedia.org/wiki/SQL:2016) support for JSON 
+ functions resolves a large number of issues reported by the community over the
+ years. This includes 
+ [json_array]({{site.url}}/docs/current/functions/json.html#json-array),
+ [json_object]({{site.url}}/docs/current/functions/json.html#json-object),
+ [json_exists]({{site.url}}/docs/current/functions/json.html#json-exists),
+ [json_query]({{site.url}}/docs/current/functions/json.html#json-query), and
+ [json_value]({{site.url}}/docs/current/functions/json.html#json-value)
+ functions that were added to Trino this year.
+* The JSON output format was added to the `EXPLAIN` statement to provide
+ anonymised query plan to enable offline analysis.
+* It became possible to comment on tables, columns of tables, and even views for
+ various connectors. Support for setting comments on views was just introduced
+ recently and includes support for Hive and Iceberg. Table comments came to
```

Comment:
We can drop the list of connectors imho

Reply:
fair...comments aren't mind blowing features..just nice